server/events/project_command_pool_executor.go

```diff
@@ -81,6 +82,10 @@ func runProjectCmdsParallelGroups(
 	for _, group := range groups {
 		res := runProjectCmdsParallel(group, runnerFunc, poolSize)
 		results = append(results, res.ProjectResults...)
+		if res.HasErrors() && group[0].AbortOnExcecutionOrderFail {
```

Comment:
How come it only checks the first one? Why not the current one?

Reply:
The AbortOnExcecutionOrder is a repo config, so it is applied on all commands. And I'm not sure I understand what you are referring to with the `current one` as we are passing groups to the function `runProjectCmdsParallel`.

Reply:
I think i mosunderstood `group[0]`

Ty